public/js/monitor.js

```diff
@@ -1,4 +1,56 @@
 "use strict";
+/* global ga */
+
+function sendEvent(category, action, label){
+  if(ga){
+  ga("send", "event", category, action, label);
+  }
+}
+
+function handleSignUpClicks(string){
+  const eventCategory = "Sign Ups";
+  const eventAction = "Clicked Sign Up";
+  if(window.location.href.indexOf("breach") > -1){
```

Comment:
What is this meaning to do?
Are you trying to check for `?breach=Adobe` style things in the URL? If so, should we use something more specific, like `window.location.search.indexOf("breach=")`?

```js
console.log(window.location.href); // "https://fx-breach-alerts.herokuapp.com/?breach=Adobe"
console.log(window.location.search); // "?breach=Adobe"
```

Or even something similar to this oddity (per [`URLSearchParams#has()`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/has)):

```js
if (new URLSearchParams(window.location.search.substring(1)).has("breach")) {...}
```